### ISSUE TITLE
`torchvision_nightly` is removed from PypI, but require for fastai=1.0.4

### ISSUE THREAD

**Issue by WillCao68** [magarveylab/NLPPrecursor#5]
Hello developers.  The version you asked for fastai requires the installation of torchvision_nightly. But torchvision_nightly is not available anymore. What is your work around now?

**Comment by WillCao68** [comment]
@nishanthmerwin can you help? I think the previous issue (issue #4 ) posted by someone else has the same issue I posted here.

**Comment by nishanthmerwin** [comment]
Thanks for bringing this to my attention + issue #4.
I'm able to reproduce the issue, but haven't found a suitable solution yet. I'll update this thread once I do!

**Comment by nishanthmerwin** [comment]
My latest commit should solve all of these issues. I ended up forking `fastai`, editing and installing from there. I also updated the instructions in the readme.

For now, this should work. I tested this locally and it seems to work fine.

```
conda create --name deepripp python=3.7
conda activate deepripp
pip install git+https://github.com/nishanthmerwin/fastai.git@deepripp_install
pip install git+https://github.com/magarveylab/nlpprecursor
```

I updated the README as well, but to use the pretrained models, be sure to use the lines. 
```
import nlpprecursor
import sys
sys.modules['protai'] = nlpprecursor
```

I'm going to close this issue and #4 for now, but if anyone has further concerns feel free to open an issue!
